src/Provider/Geonames/Geonames.php

```diff
@@ -92,6 +98,73 @@ public function reverseQuery(ReverseQuery $query): Collection
     }
 
     /**
+     * @param string|null $country
+     * @param string|null $locale
+     * @param int|null    $maxRows
+     * @param int|null    $startRow
+     * @return AddressCollection
+     * @throws \Geocoder\Exception\Exception
+     */
+    public function getCountryInfo(string $country = null, string $locale = null, int $maxRows = null, int $startRow = null): AddressCollection
```

Comment:
If `$country` is left out, then Geonames returns all the countries (250 items as of now). Let me know if this is fine or I will update it to a necessary param!